File: PP0602A_parzyste_nieparzyste/PP0602A.cpp
/* https://pl.spoj.com/problems/PP0602A/ */
#include <iostream>

using namespace std;

int main(){
    int t, i, j, n;
    int tab[100];

    cin>>t;
    while(t--){
        cin>>n;
        for(i=0;i<n;i++){
            cin>>tab[i];
        }
        for(i=1;i<n;i+=2){
            cout<<tab[i]<<" ";
        }
        for(i=0;i<n;i+=2){
            cout<<tab[i]<<" ";
        }
        cout<<endl;
    }
    return 0;
}

File: PP0602A_parzyste_nieparzyste/PP0602A.py
# https://pl.spoj.com/problems/PP0602A/
t = int(input())

while t>0:
    line = input().split()
    n= line[0]
    for i in range(2,n, 2):
        print(line[i], end=" ")
    
    for i in range(1,n, 2):
        print(line[i], end=" ")
    print()
    t=t-1